### DISCUSSION TITLE
t5x in Colab

### DISCUSSION THREAD

**Discussion by agoliaei** [google-research/t5x#984] [Q&A]
Hi, 
I understand that according to the intro readme "currently the [default public Colab](https://colab.research.google.com/) cannot be easily used to run t5x models."
Does this mean it is impossible to use Colab's TPUs for t5x training/testing? Or the setting up just takes longer time?
If the latter, is there any instruction available somewhere for this?
I appreciate your help and thanks for sharing t5x with the community.

**Comment by orleg01** [comment]
i had the same problem.

first download the git library

git clone --branch=main https://github.com/google-research/t5x
cd t5x

and then run python setup.py install
